content/hi/docs/test.md

```diff
@@ -0,0 +1,354 @@
+---
+शीर्षक: दस्तावेज़ स्मोक परीक्षण पृष्ठ
+main_menu: false
+---
+
+यह पृष्ठ दो उद्देश्यों को पूरा करता है:
+
+- यह दर्शाता है कि कुबेरनेट्स दस्तावेज़ीकरण मार्कडाउन का उपयोग कैसे करता है
+- यह एक "स्मोक परीक्षण" दस्तावेज़ प्रदान करता है जिसका उपयोग हम HTML, CSS और टेम्पलेट परिवर्तनों का परीक्षण करने के लिए कर सकते हैं जो समग्र दस्तावेज़ीकरण को प्रभावित करते हैं।
+
+## शीर्षक स्तर
+
+ऊपर दिया गया शीर्षक H2 है। पृष्ठ शीर्षक H1 के रूप में प्रस्तुत होता है। निम्नलिखित अनुभाग H3-H6 दिखाते हैं।
+
+### H3
+
+यह एक H3 अनुभाग में है।
+
+#### H4
+
+यह एक H4 अनुभाग में है।
+
+##### H5
+
+यह एक H5 अनुभाग में है।
+
+###### H6
+
+यह एक H6 अनुभाग में है।
+
+
+
+## इनलाइन तत्व
+
+इनलाइन तत्व पैराग्राफ, सूची आइटम, चेतावनी, या अन्य ब्लॉक-स्तरीय तत्व के पाठ के भीतर दिखाई देते हैं।
+
+लोरम इप्सम डोलर सिट अमेट, कोंसेकटेटूर एडिपिसिंग एलिट, सेड डू एइज्मोड टेम्पोर इंक सीडिडंट यूट लेबर एट डोलोर मैग्ना अलीक्वा.  उट एनिम एड मिनिम वेनियाम, क्विस नोस्ट्रुड एक्सरसाइजेशन उलैमको लेबरिस निसी यूट एलीक्विप एक्स ईए कॉमोडो कॉन्सेक्वाट. डुइस ऑटे इरूरे डोलोर इन रिप्रोहेन्डरिट इन वोल्प्टेट वेलेट एसे सिलम डोलोर यू फुगियात नुला पारियाटुर. एक्ससेप्टर सिंट ओकेकैट कपिडेटेट नॉन प्रोइडेंट, सुंट इन कूपा क्वि ऑफिशिया डेसेरंट मोलिट एनिम इद एस्ट लेबरम।
+
+### इनलाइन टेक्स्ट शैलियाँ
+
+- **बोल्ड**
+- _इटैलिक_
+- ***बोल्ड इटैलिक***
+- ~~लाइन थ्रू~~
+- <u>आ रेखांकित</u>
+- _<u>आ रेखांकित इटैलिक</u>_
+- **<u>आ रेखांकित बोल्ड</u>**
+- ***<u>आ रेखांकित बोल्ड इटैलिक</u>***
+- `मोनोस्पेस टेक्स्ट`
+- **`मोनोस्पेस बोल्ड`**
+
+## सूची
+
+मार्कडाउन के पास सूचियों को संसाधित करने के बारे में सख्त नियम नहीं हैं।  जब हम जेकिल से ह्यूगो में गए, तो हमने कुछ सूचियों को तोड़ दिया। उन्हें ठीक करने के लिए, निम्नलिखित बातों को ध्यान में रखें:
```

Comment:
जब हम जेकिल से ह्यूगो में गए - I don't think this translates correctly to when we moved from Jekyll to Hugo. You might need to use a different verb to convey the meaning instead of move.
```suggestion
मार्कडाउन के पास सूचियों को संसाधित करने के बारे में सख्त नियम नहीं हैं। जब हम जेकिल से ह्यूगो में गए, तो हमने कुछ सूचियों को तोड़ दिया। उन्हें ठीक करने के लिए, निम्नलिखित बातों को ध्यान में रखें:
```

Reply:
A suggestion: स्थानांतरित to me sounds better.
"जब हम जेकिल से ह्यूगो पर स्थानांतरित किये"